a workaround until we fix our llvm to properly report
__builtin_cpu_crap as not supported

--- a/driver/others/dynamic_power.c
+++ b/driver/others/dynamic_power.c
@@ -216,7 +216,7 @@ static int cpuid(void)
 #define __has_builtin(x)   0
 #endif
 
-#if defined(_AIX) || !__has_builtin(__builtin_cpu_is)
+#if 1
 static int __builtin_cpu_is(const char *arg)
 {
     static int ipinfo = -1;
@@ -241,7 +241,7 @@ static int __builtin_cpu_is(const char *arg)
 }
 #endif
 
-#if defined(_AIX) || !__has_builtin(__builtin_cpu_supports)
+#if 1
 static int __builtin_cpu_supports(const char *arg)
 {
     return 0;
